course/page/choice.py

```diff
@@ -65,20 +64,78 @@ def markup_to_html_plain(page_context, s):
     return s
 
 
-# {{{ choice question base
+# {{{ choice data model
 
-class ChoiceQuestionBase(PageBaseWithTitle, PageBaseWithValue):
+class ChoiceModes:
+    INCORRECT = "incorrect"
+    CORRECT = "correct"
+    DISREGARD = "disregard"
+    ALWAYS_CORRECT = "always_correct"
```

Comment:
Can you explain the use case of "aways_correct"? Thanks.

Reply:
Sure. It exists to ensure fair scoring of a multi-select question in which one option has turned out to be flawed. The net effect of `disregard` is to score the question as if that option didn't exist. But some students may have received points from the broken option, so `disregard` would take those points away. Cue lots of (somewhat justified) complaints from grumpy students. `always_correct` prevents that by grading any answer as a correct one, therefore never leading to a point decrease.

If this explanation was helpful, could you copy-paste it to the docs? Thanks! :)

Reply:
Ok. Thanks.

Reply:
I've submit a PR in https://github.com/inducer/relate-sample/pull/10/files, can you review that?